packages/terra-site/src/examples/list/MultiSelectMaxSelection.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import MultiSelectList from 'terra-list/lib/MultiSelectList';
 
 const list = () => (
-  <MultiSelectList isDivided maxSelectionCount={2}>
+  <MultiSelectList maxSelectionCount={2}>
```

Comment:
I'll bring it as a war room topic. This by all means does not block this pull request.